site/content/docs/5.3/components/navbar.md

```diff
@@ -339,7 +339,7 @@ Mix and match with other components and utilities as needed.
 **New in v5.2.0:** Navbar theming is now powered by CSS variables and `.navbar-light` has been deprecated. CSS variables are applied to `.navbar`, defaulting to the "light" appearance, and can be overridden with `.navbar-dark`.
 {{< /callout >}}
 
-Navbar themes are easier than ever thanks to Boosted's combination of Sass and CSS variables. The default is our "light navbar" for use with light background colors, you can also apply `.navbar-dark` for dark background colors. Then, customize with `.bg-*` utilities.
+Navbar themes are easier than ever thanks to Boosted's combination of Sass and CSS variables. The default is our "light navbar" for use with light background colors, but you can also apply `data-bs-theme="dark"` to the `.navbar` parent for dark background colors. Then, customize with `.bg-*` utilities.
```

Comment:
Is not true atm (border-color, orange used, ...).
Does it need to be mentionned in the migration guide or not ?
Do we need to track this kind of changes inside #1776 ?

Reply:
There's no need to mention anything in the migration guide for now IMO. But yeah I'll need to track this paragraph in the tracking issue to change it or modify it if we don't integrate finally the dark mode.

Reply:
✅ Added a reference to your comment in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/issues/1776